src/lib/y2storage/proposal/lvm_helper.rb

```diff
@@ -166,9 +167,9 @@ def new_volume_group
       # @return [Boolean]
       def try_to_reuse?
         # Setting introduced to completely avoid LVM reusing in D-Installer.
-        # It's a new playground, so no need to carry YaST behaviors that have
-        # proven to be confusing.
-        return false if settings.lvm_vg_reuse == false
+        # However it happened to be handy even in old yast later on. Preset
+        # settings can get overwritten by Linuxrc boot param.
+        return false if StorageEnv.instance.reuse_lvm? == false || settings.lvm_vg_reuse == false
```

Comment:
I would not short-circuit things here. We already have a well defined `settings.lvm_vg_reuse` that should be honored. This makes things hard to debug (inconsistency between the setting and the behavior).

What I would do instead is relying on `StorageEnv.instance.reuse_lvm?` to set the default value of `settings.lvm_vg_reuse`. Unsurprisingly defaults are applied at `ProposalSettings#apply_defaults`.

Reply:
I've relocated of ```lvm_vg_reuse``` default setup.

Idea is that it should
1) be set by default (```DEFAULT```) to true or
2) set according to what is in control file (added in this PR, if we decide to use it in the future) or
3) set it to value explicitly requested by a user if user set something, keep it as is if user set nothing